src/openrct2/windows/view_clipping.c

```diff
@@ -0,0 +1,285 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
@janisozaur Do you want `2017` for new files, or would you prefer to keep it consistent?